about/triage-meeting.md

```diff
@@ -1,14 +1,37 @@
-The compiler **triage meeting** is the place for tracking regressions
-and progress on urgent bugs. The meeting takes place every week.  It
-is currently scheduled for Fridays at 10am Boston time (note that we
-track the US when it comes to Daylight Savings Time). There is also a
-calendar invite, you can ping `@nikomatsakis` to be added.
+# Triage Meeting
+The compiler **triage meeting** is the place for tracking regressions, progress on urgent bugs and
+checking-in with the compiler team's working groups. Each meeting, two working groups (rotating)
+check-in with their progress and blockers.
 
-[internals]: https://internals.rust-lang.org/t/compiler-steering-meeting/8588
+The meeting takes place every week. See the [compiler team calendar](../README.md#meeting-calendar)
+for the next scheduled meeting (normally weekly).
 
-The triage meeting takes place on [Zulip](chat-platform.md).  Look for
-a topic like "#54818 weekly meeting YYYY-MM-DD" or "weekly meeting
-YYYY-MM-DD" (with the proper date, of course).
+The triage meeting takes place on [Zulip](chat-platform.md). Look for a topic like
+"#54818 weekly meeting YYYY-MM-DD" or "weekly meeting YYYY-MM-DD" (with the proper date, of course).
 
+## Working group check-in
+This section contains the scheduled check-ins for working groups:
 
+- **2019-03-14:** [wg-rls-2.0], [wg-meta]
+- **2019-03-21:** [wg-self-profile], [wg-rfc-2229]
+- **2019-03-28:** [wg-self-profile], [wg-rfc-2229]
```

Comment:
Nit: I expect this to be sorted in "most recent to last" -- I'd probably just make one big list?

I also think that if we should adopt a convention in the working-groups directory like

```
working-group/wg-foo/updates/2019.03.14.md
```

which would contain the update? Maybe? (Or maybe it's too much)

The idea is though that (a) if you are browsing a wg you can quickly skim the last few updates and (b) we can link to them from there. 

Anyway doesn't have to be part of this PR.

Reply:
I've updated the ordering.